x-pack/plugins/cloud_security_posture/public/common/api/use_latest_findings_data_view.ts

```diff
@@ -62,15 +62,19 @@ export const useLatestFindingsDataView = (dataView: string) => {
     }
```

Comment:
btw what `use perfected kibana data views` means (in the TODO for this hook)? not directly related to this PR, but I'd add more context to this todo, as I for instance wouldn't know what to do with it